src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputPackageDiffDirectory.java

```diff
@@ -0,0 +1,8 @@
+package com.puppycrawl.tools.checkstyle.checks;
+
+public class InputPackageDiffDirectory {
```

Comment:
What is the new input required for ? How is it supposed to be different from `InputPackageDiffDirectory` ?